Radio.m

```diff
@@ -986,6 +993,131 @@ - (void) infoResponseCallback:(NSString *)cmdResponse {
 }
 
 
+/**
+ * Process a response from a Version command
```

Comment:
This is really picky - sorry, but please replace this style of comment with

//
// Comment
//

And not use /* */ comment delimiters - this is for consistency with all the other comments